packages/jazz-nodejs/src/index.ts

```diff
@@ -61,5 +74,15 @@ export async function startWorker<Acc extends Account>({
     context.done();
   }
 
-  return { worker: context.account as Acc, done };
+  const inboxPublicApi = {
+    subscribe: inbox.subscribe.bind(inbox) as Inbox["subscribe"],
+  };
```

Comment:
Reducing the API surface to limit the possible operations on Inbox only to subscribe